propertyvalueestimate.py

```diff
@@ -0,0 +1,51 @@
+from selenium.webdriver.chrome.options import Options
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from bs4 import BeautifulSoup
+import domainapi as api
+
+
+addressId = 2017019671
+def getPropertySoup(addressId, headless=False):
+
+    url = "https://www.domain.com.au/" + addressId
```

Comment:
put this url in the main file with the other urls, DOMAIN_CLIENT_URL, and name the URL_ENDPOINT -> DOMAIN_API_V1_URL